Script/AtomicEditor/ui/frames/MainFrame.ts

```diff
@@ -111,12 +111,18 @@ class MainFrame extends ScriptWidget {
         if (this.menu.handlePopupMenu(target, refid))
             return true;
 
+        if (this.openedMenu && this.openedMenu.id == target.id + " popup") {
+            this.openedMenu.close();
+            this.openedMenu = null;
+            return true;
+        }
+
```

Comment:
hm, do you think that it's necessary to release the openedMenu in that case? It will be overwritten on the next menu click, so it wouldn't cause any mistakes